alerting/src/main/kotlin/org/opensearch/alerting/util/destinationmigration/DestinationMigrationCoordinator.kt

```diff
@@ -45,7 +45,6 @@ class DestinationMigrationCoordinator(
     }
 
     override fun clusterChanged(event: ClusterChangedEvent) {
-        logger.info("Detected cluster change event for destination migration")
```

Comment:
Should this be swapped to debug instead of removed?

Reply:
We can, but it's redundant. We already have an if-else clause for each condition, which captures the required information in the log lines.